### ISSUE TITLE
Snapshot Release

### ISSUE THREAD

**Issue by Fabio1988** [openstack4j/openstack4j#195]
Hey guys :)

This repo has now new commits, I am really looking forward to that because there were many vulnerabilities... And I wanna give the dev snapshot a try.

How can I use the snapshot release? I tried different ways but somehow I can't find a updated version in [Snapshots](https://oss.sonatype.org/content/repositories/snapshots).... [Snapshots](https://oss.sonatype.org/content/repositories/snapshots/com/github/openstack4j/)

When is next release planned? :) @olivergondza

**Comment by Fabio1988** [comment]
at least I know now that the snapshot is not ready to build, following error:

```
[ERROR] home/openstack4j/core/src/test/java/org/openstack4j/test/common/MicroVersionTest.java:[4,30] cannot access org.testng.annotations.DataProvider
  bad class file: /home/server/.m2/repository/org/testng/testng/7.7.0/testng-7.7.0.jar(org/testng/annotations/DataProvider.class)
    class file has wrong version 55.0, should be 52.0
    Please remove or make sure it appears in the correct subdirectory of the classpath.
```

**Comment by olivergondza** [comment]
Yup, the project did not have continuous snapshot publishing working since the fork.

**Comment by olivergondza** [comment]
I have released 3.11 just now.

**Comment by Fabio1988** [comment]
> I have released 3.11 just now.

I assume that was not successful?! :)
I will wait for next week, really looking forward to it

**Comment by Fabio1988** [comment]
> Yup, the project did not have continuous snapshot publishing working since the fork.

Would be nice to enable that workflow again :)
